src/main/java/org/folio/circulation/domain/FeeFineAction.java

```diff
@@ -22,8 +22,8 @@ public Double getBalance() {
     return representation.getDouble("balance");
   }
 
-  public Double getAmountAction() {
-    return representation.getDouble("amountAction");
+  public FeeAmount getAmountAction() {
+    return FeeAmount.from(representation, "amountAction");
   }
```

Comment:
this is fee amount (or part of the amount) that was refunded/paid/transferred/waived/etc. during a single transaction.  

For example: there is 50$ fee amount, patron pays 25$ - in this case `amount action` is 25$

Reply:
Hmm, I don't think I follow the example. I'm going to share my own example to try to check my understanding.

Given an overdue fee of $50 has been charged to a patron
When the patron pays $30 towards it
Then is the `amount action` $30 because that was the amount relevant to the action?
Or is the `amount action` $20 because that is the outstanding balance of the fee?

Reply:
The first one.

Reply:
In which case, given that it is part of the action, could this be called `amount` rather than `action amount` (which I think is either redundant or does not make sense)?

Reply:
renamed to `amount`.